coalib/results/Diff.py

```diff
@@ -244,10 +245,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._original
 
-    @property
-    def modified(self):
+    def _raw_modified(self):
```

Comment:
So we would copy your new `modified` code to `__init__`, initially linebreak-ify the originally and all patches are referring to this version then. We would need to patch `add_line`, `change_line` and so on to properly convert inputs then^^
What do you think?